integration-templates/ashby/nango.yaml

```diff
@@ -1,29 +1,43 @@
 integrations:
   ashby:
-    ashby-candidates:
-      runs: every hour
-      returns:
-        - AshbyCandidate
-      description: |
-        Fetch a list of all candidates from your ashby account.
-        Details: Increamental sync, doesn't track deletes, metadata is required.
-    ashby-jobs:
-      runs: every hour
-      returns:
-        - AshbyJob
-      description: |
-        Fetch a list of all jobs from your ashby account.
-        Details: Increamental sync, doesn't track deletes, metadata is required.
-    ashby-create-application:
-      type: action
-      returns: AshbyCreateApplicationResponse
-      description: |
-        Action to consider a candidate for a job.
-    ashby-create-note:
-      type: action
-      returns: AshbyCreateNoteResponse
-      description: |
-        Action to create a note on a candidate.
+    syncs:
+      ashby-candidates:
+        runs: every hour
+        output: AshbyCandidate
+        description: |
+          Fetches a list of all candidates from your ashby account
+        metadata:
+          description: |
+            Since Ashby doesn't use a lastSyncDate value to increment its sync,
+            we need to store a candidatelastsyncToken which will be used to increment the candidates syncs
+          scopes: candidatelastsyncToken
+        sync_type: incremental
+        endpoint: /ashby/candidates
+      ashby-jobs:
+        runs: every hour
+        output: AshbyJob
+        description: |
+          Fetches a list of all jobs from your ashby account
+        metadata:
+          description: |
+            Since Ashby doesn't use a lastSyncDate value to increment its sync,
```

Comment:
This is a weird description. I think a more general sentence before this caveat would make sense